doc/architecture/sensor_model.adoc

```diff
@@ -0,0 +1,6 @@
+= Sensor model
+
+Sensor models consume sensor-view data and produce sensor-data data.
```

Comment:
```suggestion
Sensor models consume and produce sensor-view messages.
```

Reply:
Exchanged "data" for "messages".

I recommend to keep the same sentence structure ("X consumes Y and produces Z") across all model-type topics, even if Y and Z are the same.
IMO this makes clearer, that there are inputs and outputs.